docs/02-Install.md

```diff
@@ -1,13 +1,111 @@
 # 2.0 - Install
 
-## 2.1 Dependencies
+## 2.1 Dependencies and installation
 
-## 2.2 Setup
+Dependencies and installation information are documented on the LORIS-MRI
+  [README.md](../README.md) file.
 
-## 2.3 Configuration
 
-## 2.4 Pipeline flow
+## 2.2 Configuration
 
-## 2.5 Common errors
+**NEED TO TALK ABOUT THE CREATION OF NIFTI OPTIONS AS IT IS REFERENCED IN THE
+  PIPELINE FLOW DOCUMENTATION.**
 
-[FAQ](AppendixA-FAQ.md)
+## 2.3 Pipeline flow
+
+By default the pipeline is designed for **raw DICOM MRI data**, collected by a
+  longitudinally-organized multisite study with a defined data acquisition
+  protocol. With modifications and further customization, it can handle any
+  modality of imaging data.
+
+Default images insertion into LORIS follows the following steps:
+
+1. Upload of a zip of the DICOM study via the **Imaging Uploader** module (or
+     DICOM study transfer directly on the server hosting LORIS).
+2. DICOM insertion into the `tarchive` tables in order to be able to see the
+     information stored in the DICOMs via the **DICOM Archive** module.
+3. Conversion of the DICOMs into MINC files for protocol validation and
+     insertion into the LORIS database. By default it will also create NIfTI
+     images from the MINC files and insert them into LORIS. See the
+     [Configuration](#2.2-configuration) section above if you want to disable
+     this option.
+     
+    a. If a scan matches one of the protocol defined in the `mri_protocol`
+         table, then the MINC image will be stored into the `files` tables and
+         can be visualized in the **Imaging Browser** module and
+         **BrainBrowser**.
+
+    b. If a scan does not match any of the protocol defined in the
+         `mri_protocol` table, then the MINC image of this scan will be stored
```

Comment:
I would say a record of that MINC image will be stored/accessed via the MRI Violated scans module with a hyperlink to navigate it through BrainBrowser.